microbosh-setup/micro_cf.yml

```diff
@@ -0,0 +1,375 @@
+# The following line helps maintain current documentation at http://docs.cloudfoundry.org.
+# code_snippet cf-minimal-azure start
+---
+name: cf
+director_uuid: bfb078f7-99bc-4199-aecc-9fbb99f526b3 # Replace it with the UUID of your MicroBosh
+
+releases:
+- {name: cf, version: latest}
+
+networks:
+- name: cf_private
+  type: manual
+  subnets:
+  - range: 10.0.16.0/20
+    gateway: 10.0.16.1
+    dns: [10.0.0.4]  # Replace it with your own dns ip 
+    reserved: ["10.0.16.2 - 10.0.16.3"]
+    static: ["10.0.16.4 - 10.0.16.10"]
+    cloud_properties:
+      virtual_network_name: #vnet#
+      subnet_name: #cfsubnet#
+      tcp_endpoints:
+      - "22:22"
+      - "80:80"
+      - "443:443"
+      - "4222:4222"
+      - "4443:4443"
+      - "6868:6868"
+      - "25250:25250"
+      - "25555:25555"
+      - "25777:25777"
+      udp_endpoints:
+      - "53:53"
+      - "68:68"
+
+- name: reserved
+  type: vip
+  cloud_properties: {}
+
+resource_pools:
+- name: large_z1
+  network: cf_private
+  stemcell:
+    name: bosh-azure-hyperv-ubuntu-trusty-go_agent
+    version: '0000'
+  cloud_properties:
+    instance_type: Standard_A4
+
+compilation:
+  workers: 5
+  network: cf_private
+  reuse_compilation_vms: true
+  cloud_properties:
+    instance_type: Standard_A1
+
+update:
+  canaries: 1
+  max_in_flight: 1
+  serial: false
+  canary_watch_time: 30000-600000
+  update_watch_time: 5000-600000
+
+jobs:
+- name: cf_z1
+  instances: 1
+  persistent_disk: 102400
+  resource_pool: large_z1
+  templates:
+  - {name: nats, release: cf}
+  - {name: nats_stream_forwarder, release: cf}
+  - {name: metron_agent, release: cf}
+  - {name: etcd, release: cf}
+  - {name: etcd_metrics_server, release: cf}
+  - {name: debian_nfs_server, release: cf}
+  - {name: postgres, release: cf}
+  - {name: cloud_controller_ng, release: cf}
+  - {name: cloud_controller_worker, release: cf}
+  - {name: cloud_controller_clock, release: cf}
+  - {name: nfs_mounter, release: cf}
+  - {name: haproxy, release: cf}
+  - {name: hm9000, release: cf}
+  - {name: doppler, release: cf}
+  - {name: loggregator_trafficcontroller, release: cf}
+  - {name: login, release: cf}
+  - {name: uaa, release: cf}
+  - {name: gorouter, release: cf}
+  - {name: dea_next, release: cf}
+  - {name: dea_logging_agent, release: cf}
+  - {name: collector, release: cf}
+  networks:
+  - name: cf_private
+    static_ips: [10.0.16.4]
+    default: [gateway, dns]
+  - name: reserved
+    static_ips: #cfip#
+  update:
+    serial: true
+  properties:
+    etcd_metrics_server:
+      nats:
+        machines: [10.0.16.4]
+        password: c1oudc0w
+        username: nats
+    nfs_server:
+      address: 10.0.16.4
+      allow_from_entries: [10.0.16.0/20]
+    ha_proxy:
+      ssl_pem: |
+        -----BEGIN CERTIFICATE-----
```

Comment:
I have double checked with abel, he said this ssl is common to bosh development. I have his promise , so it's ok to check in them @AbelHu